### ISSUE TITLE
Allow login to management interface to persist

### ISSUE THREAD

**Issue by vrillusions** [syncthing/syncthing#3334]
A while back it was changed to require a login to access management interface.  While the benefits are obvious a big downside is when I open up all my nodes in tabs and have to login to each of them and have to do this any time I want to access them.  My current workaround has been to go in to advance settings and disable that option so it still doesn't ask for a login as I'm confident enough that no one is going access it and also none of it is super sensitive information.

What I propose is something akin to the "remember me" option when logging in to websites.  ie I open management interface it asks for login and then I can check a box that says to remember me.  This creates a secure cookie that persists in browser.  Biggest hurdle I see for this is the use of Basic Auth window for logins currently.

**Comment by AudriusButkevicius** [comment]
Most browsers remember basic auth.
Alternatively, you can embed the authentication in the URL.

**Comment by vrillusions** [comment]
Yeah I have mine set not to.  I tried whitelisting cookies to see if that would fix it (by default I clear cookies on close unless whitelisted) and that didn't work either.  I'll have to look at the authentication in url though which should be fine since the request is sent after the tls connection is brought up, I think.

**Comment by calmh** [comment]
We do have a session cookie, so making it survive longer would be doable.
